### ISSUE TITLE
Wireless Crafting Terminal doesn't work

### ISSUE THREAD

**Issue by Horf** [DimensionalDevelopment/VanillaFix#140]
Wireless Crafting Terminal (https://minecraft.curseforge.com/projects/wireless-crafting-terminal - 1.12.2-3.11.86)
doesn't work, while VanillaFix (1.0.10-99) is active (on Client)
VanillaFix (same version) on Server only works fine.

**Comment by ZombieHDGaming** [comment]
We need a little more info than it doesn't work. We need details, what were you doing while this was happening, is there an error in the log, etc. We can't help if we don't have the right stuff.

**Comment by Runemoro** [comment]
This is a VanillaFix feature. It's warning you about a possible problem with one of your mods that you should report to them. If you uninstall VanillaFix, the problem is still there but it just won't warn you about it. You can also disable the warning VanillaFix's config.

**Comment by Runemoro** [comment]
Oh, wait, is it just the error message or is the wireless crafting terminal broken with VanillaFix and works without?

**Comment by Horf** [comment]
Hm, seems to be a problem with WCT itself. There was an update to fix a similar crash

Still curious, why it worked without vanillafix^^

**Comment by Runemoro** [comment]
Did you test it more than twice with and without VanillaFix? It could have been a coincidence.

**Comment by Horf** [comment]
Yeah...there was a time, where I could play 30 minutes with vanillafix and wct worked fine and then there was this error again.
Tested it on different clients, too.

Only person who never had problems with this and didn't know there was something about it, was the one who never had vanillafix installed.
